security/advancedtls/crl.go

```diff
@@ -516,3 +516,34 @@ func extractCRLIssuer(crlBytes []byte) ([]byte, error) {
 	}
 	return issuer, nil
 }
+
+func hasExpired(crl x509.RevocationList, now time.Time) bool {
```

Comment:
It's probably better to pass the pointer here, assuming x509.RevocationList is a sizable struct.

Reply:
Good call